modules/ROOT/pages/alerts.adoc

```diff
@@ -2,32 +2,90 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: alerts-using.adoc, alerts-config.adoc
 
 Alerts indicate whether a resource (such as a Mule app) is behaving as expected or exceeding a defined threshold.
 
-You can implement any of several types of alerts. From the *Alerts* page in Anypoint Monitoring, you can create xref:alerts-config.adoc#basic-alerts[basic alerts], or you can link to alerts for API Manager and Runtime Manager alerts.
-If you have a Titanium subscription, you can also view, enable, and disable xref:alerts-config.adoc#advanced-alerts[advanced alerts] that are configured for graphs in custom dashboards.
+You can implement any of several types of alerts. From the *Alerts* page in Anypoint Monitoring, you can create xref:basic-alerts.adoc[basic alerts], or you can link to alerts for API Manager and Runtime Manager alerts.
+If you have a Titanium subscription, you can also view, enable, and disable xref:advanced-alerts.adoc[advanced alerts] that are configured for graphs in custom dashboards.
 
-// should the below information remain here? Or should we redo this when we address Trust?
+Using Anypoing Monitoring, you can configure two different types of alerts:
 
-When you configure an advanced alert for a graph, an alert status icon appears at the top-left corner of the graph. For example, the green dot in the bell indicates that the status is `OK`:
+* xref:basic-alerts.adoc[Basic alerts] for servers and Mule apps
++
+Limit per organization: Up to 50 basic alerts for users who do not have a Titanium subscription to Anypoint Platform. +
+If you have a Titanium subscription, the limit is 100 basic alerts, and the limits increase as you purchase more vCores. For complete details about subscriptions, contact your customer support representative. See also https://www.mulesoft.com/anypoint-pricing[Anypoint Platform pricing].
+* xref:advanced-alerts.adoc[Advanced alerts] for graphs in custom dashboards in Anypoint Monitoring
++
+Limit per organization: Up to 20 advanced alerts.
++
+Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the *General* tab for the graph. If the value passes the threshold, the advanced alert is triggered. The advanced alert is triggered only if the alert state changes (from `OK` state to `Alerting` state or vice versa).
+* Operational alerts for APIs through API Manager
+* Operational alerts for the runtime through Runtime Manager
```

Comment:
We can't reproduce limits for other products' alerts in case they change. As for merging limits, I will reevaluate that.

I think Mule runtime engine is right.

Reply:
```suggestion
* Operational alerts for APIs through API Manager
* Operational alerts for server and app events through Runtime Manager
```